packages/grafana-toolkit/src/cli/tasks/package.build.ts

```diff
@@ -43,6 +43,21 @@ export const savePackage = useSpinner<SavePackageOptions>(
 const preparePackage = async (pkg: any) => {
   pkg.main = 'index.js';
   pkg.types = 'index.d.ts';
+
+  const version: string = pkg.version;
+  const name: string = pkg.name;
+  const deps: any = pkg.dependencies;
+  console.log('PREPARE', pkg.name, version);
+  if (name.endsWith('/ui')) {
+    deps['@grafana/data'] = version;
+  } else if (name.endsWith('/runtime')) {
+    deps['@grafana/data'] = version;
+    deps['@grafana/ui'] = version;
+  } else if (name.endsWith('/toolkit')) {
+    deps['@grafana/data'] = version;
+    deps['@grafana/ui'] = version;
+  }
+
```

Comment:
@dprokop -- thoughts on this approach?   Is this the right place?

Reply:
I think it's fine as it's saved directly to dist and the original package.json ain't modified.